toast-tk-runtime/src/main/java/io/toast/tk/runtime/report/TemplateHelper.java

```diff
@@ -9,7 +23,9 @@
 import io.toast.tk.dao.domain.impl.test.block.line.TestLine;
 
 public class TemplateHelper {
-
+	
+	private static int HTML_LENGTH_MAX = 40;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Rename this field "HTML_LENGTH_MAX" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3008)